access/management/commands/pregenerate_exercises.py

```diff
@@ -4,7 +4,8 @@
 from django.core.management.base import BaseCommand, CommandError
 from access.views import config
 from util.personalized import delete_pregenerated_exercise_instances, \
-    prepare_pregenerated_exercises_directory, generate_exercise_instances
+    prepare_pregenerated_exercises_directory, generate_exercise_instances, \
+    pregenerated_exercise_instances
```

Comment:
Import could be reformatted to

```python
from foobar import (
  baz1,
  baz2,
  baz3,
)
```
comma in every line and words in sorted order.